go/vt/vtgr/controller/group.go

```diff
@@ -31,22 +33,31 @@ import (
 var (
 	groupOnlineSize = stats.NewGaugesWithMultiLabels("MysqlGroupOnlineSize", "Online MySQL server in the group", []string{"Keyspace", "Shard"})
 	isLostQuorum    = stats.NewGaugesWithMultiLabels("MysqlGroupLostQuorum", "If MySQL group lost quorum", []string{"Keyspace", "Shard"})
+
+	heartbeatThreshold = flag.Int("group_heartbeat_threshold_sec", 0, "VTGR will trigger backoff on inconsistent state and the group heartbeat staleness exceeds the threshold here. Need to set together with -enable_heartbeat_check")
```

Comment:
```suggestion
	heartbeatThreshold = flag.Int("group_heartbeat_threshold", 0, "VTGR will trigger backoff on inconsistent state if the group heartbeat staleness exceeds this threshold (in seconds). Should be used along with -enable_heartbeat_check")
```

Reply:
I prefer the old flag name to this one. Adding "(in seconds)" should help.
Sorry to nitpick on this, but flags are painful to change, and together with their description are part of the interface we present to users, so I tend to be picky about them.